packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -120,10 +120,22 @@ interface FluidDataStoreMessage {
 	type: string;
 }
 
+/**
+ * This version of the interface is private to this the package. it should never be exported under any tag.
+ * It is used to manage interactions between layers within the package itself. If something is needed
```

Comment:
nit: Don't use "layers" here.  Maybe this instead?
```suggestion
 * It is used to manage interactions within the container-runtime package. If something is needed
```